.github/workflows/py-workflow.yml

```diff
@@ -170,7 +170,7 @@ jobs:
       run: python3 -m pytest
 
     - name: Raise version
-      if: ${{ inputs.deploy == 'true' }}
+      if: ${{ (inputs.deploy == 'true') && (github.ref != 'refs/heads/dev') }}
```

Comment:
`inputs.deploy` is already doing the triggering, so if we do not want to deploy on actions on `dev` branches, the configuration has to be done in each repo